servicetemplates/https%3A%2F%2Fust-quantil.github.io%2Fquantum%2Fapplications%2Fservicetemplates/QHAna_w3/ServiceTemplate.tosca

```diff
@@ -0,0 +1,321 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Definitions xmlns="http://docs.oasis-open.org/tosca/ns/2011/12" xmlns:winery="http://www.opentosca.org/winery/extensions/tosca/2013/02/12" xmlns:selfservice="http://www.eclipse.org/winery/model/selfservice" xmlns:testwineryopentoscaorg="http://test.winery.opentosca.org" targetNamespace="https://ust-quantil.github.io/quantum/applications/servicetemplates" id="quantumApplicationsServiceTemplates-QHAna_w3">
+    <ServiceTemplate name="QHAna_w3" targetNamespace="https://ust-quantil.github.io/quantum/applications/servicetemplates" id="QHAna_w3">
```

Comment:
Please add an output mapping, such that the URL of the provisioned UI will be output to the user in the OpenTOSCA UI. See [an old definition](https://github.com/OpenTOSCA/tosca-definitions-public/blob/master/servicetemplates/http%253A%252F%252Fopentosca.org%252Fservicetemplates/HomeAssistant_Bare_Docker/ServiceTemplate.tosca#L8) as an example.

Reply:
I have added it